db/schema.rb

```diff
@@ -33,6 +33,39 @@
   add_index "authentications", ["uid", "provider"], :name => "index_authentications_on_uid_and_provider", :unique => true
   add_index "authentications", ["user_id", "provider"], :name => "index_authentications_on_user_id_and_provider", :unique => true
 
+  create_table "c_rater_feedback_items", :force => true do |t|
```

Comment:
FYI: Changed in https://github.com/concord-consortium/lara/commit/3f1d919d1a1b442963c9621559e116d157778d84

I was planning to implement multiple table inheritance using this gem:
https://github.com/hzamani/active_record-acts_as
as it looked simple enough. However I didn't notice that it required Rails 4.x. We could use deprecated version of this gem (ver 0.1), but probably it doesn't make sense. I haven't found reasonable alternative (yet).

So at the moment `CRater::FeedbackItem` inherits from `Embeddable::FeedbackItem`, but simply uses its own db table. I could split extra fields into separate table and create explicit association, but I'm not sure if it's worth it.